src/proto/flwr/proto/recordset.proto

```diff
@@ -47,16 +48,18 @@ message ConfigsRecordValue {
     // Single element
     double double = 1;
     sint64 sint64 = 2;
-    bool bool = 3;
-    string string = 4;
-    bytes bytes = 5;
+    uint64 uint64 = 3;
```

Comment:
@mohammadnaseri why do we need this?

Reply:
@danieljanes we are still iterating on this PR. Is it b/c IDs are available to our users. However, if users want to store these IDs inside the messages, they may get an exception saying the IDs are too large for `sint64`.